content/blog/template/index.md

```diff
@@ -0,0 +1,154 @@
+    };
+
+    // LED strip backend configuration: RMT
+    led_strip_rmt_config_t rmt_config = {
+        // Set the clock source
+        .clk_src = RMT_CLK_SRC_DEFAULT,
+        // Set the RMT counter clock
+        .resolution_hz = LED_STRIP_RMT_RES_HZ,
+        // Set the DMA feature (not supported on the ESP32-C6)
+        .flags.with_dma = false,
+    };
+
+    // LED Strip object handle
+    led_strip_new_rmt_device(&strip_config, &rmt_config, &led_strip);
+}
+
+void app_main(void)
+{
+    configure_led();
+    led_strip_set_pixel(led_strip, 0, 20, 0, 0);
+    led_strip_refresh(led_strip);
+}
+{{< /highlight >}}
+
+
+You can highlight some specific lines on your code:
+
+{{< highlight c "linenos=table,hl_lines=3 5 7 9 11" >}}
+    led_strip_config_t strip_config = {
+        // Set the GPIO8 that the LED is connected
+        .strip_gpio_num = 8,
+        // Set the number of connected LEDs, 1
+        .max_leds = 1,
+        // Set the pixel format of your LED strip
+        .led_pixel_format = LED_PIXEL_FORMAT_GRB,
+        // LED model
+        .led_model = LED_MODEL_WS2812,
+        // In some cases, the logic is inverted
+        .flags.invert_out = false,
+    };
+{{< /highlight >}}
+
+### Add YouTube videos
+
+{{< youtube D86gQ4knUnc >}}
+
+### Add Asciinema
+
+[Please see the contribution guideline](../../pages/contribution-guide/asciinema-casts)
+
+### Flow Charts
+
+[Please see the contribution guideline](../../pages/contribution-guide/content-writing-workflow)
+
+### Timeline
+
+The following table shows ESP-IDF support of Espressif SoCs where alt text and alt text denote preview status and support, respectively. The preview support is usually limited in time and intended for beta versions of chips. Please use an ESP-IDF release where the desired SoC is already supported.
+
+{{< timeline >}}
+
+{{< timelineItem icon="github" header="ESP-IDF v4.4" badge="Service" subheader="Supported SoCs" >}}
+{{< badge >}}ESP32{{< /badge >}}
+{{< badge >}}ESP32-S2{{< /badge >}}
+{{< badge >}}ESP32-C3{{< /badge >}}
+{{< badge >}}ESP32-S3{{< /badge >}}
+{{< /timelineItem >}}
+
+{{< timelineItem icon="github" header="ESP-IDF v5.0" badge="Service" subheader="Supported SoCs" >}}
+{{< badge >}}ESP32-C2{{< /badge >}}
+{{< /timelineItem >}}
+
+{{< timelineItem icon="github" header="ESP-IDF v5.1" badge="Service" subheader="Supported SoCs" >}}
+{{< badge >}}ESP32-C6{{< /badge >}}
+{{< badge >}}ESP32-H2{{< /badge >}}
+{{< /timelineItem >}}
+
+{{< timelineItem icon="github" header="ESP-IDF v5.2" badge="Service" subheader="Supported SoCs" >}}
+{{< badge >}}ESP32{{< /badge >}}
+{{< badge >}}ESP32-S2{{< /badge >}}
+{{< badge >}}ESP32-C3{{< /badge >}}
+{{< badge >}}ESP32-S3{{< /badge >}}
+{{< /timelineItem >}}
+
+{{< timelineItem icon="github" header="ESP-IDF v5.3" badge="Preview" subheader="Supported SoCs" >}}
+{{< badge >}}ESP32-P4{{< /badge >}}
+{{< badge >}}ESP32-C5{{< /badge >}}
+{{< /timelineItem >}}
+
+{{< /timeline >}}
```

Comment:
I couldn't build it locally because it fails with 

```
failed to extract shortcode: template for shortcode "timeline" not found
```

Should it be added to `assets` just like https://github.com/mfg92/hugo-shortcode-timeline/tree/main/assets/shortcode-timeline?

Reply:
Forget about it. It worked after pulling the submodules (maybe we should add this observation in the contribution guide)